.github/workflows/chaos-test.yml

```
@@ -0,0 +1,112 @@
+        
+      - name: Start application in background
+        run: |
+          npm run start:prod &
+          sleep 10
+        env:
+          NODE_ENV: test
+          DATABASE_URL: postgresql://postgres:postgres@localhost:5432/test
+          REDIS_HOST: localhost
+          REDIS_PORT: 6379
+          JWT_SECRET: test-secret
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)